client/lib/pages/onboarding/country_list_page.dart

```diff
@@ -27,30 +27,49 @@ class CountryListPage extends StatefulWidget {
 
 class _CountryListPageState extends State<CountryListPage> {
   String selectedCountryCode;
+  var listData;
```

Comment:
- Define type when giving value late for IDE Intellisense
- Use semantic name
```suggestion
  Iterable<IsoCountry> selectedCountries;
```